### PR TITLE
Wizard: Show if server asks for client certificate #2799 #69

### PR DESCRIPTION
**Pull request by guruz**
This also nicely displays the 'Untrusted domain' message of oC.
The link to add a trusted domain (via web browser) is clickable.

### CODE REVIEW

**Comment by guruz** [comment]
@ogoffart Please re-evaluate. I extract the message now early so reply is not deleted before.
I also think the sync dialog is ok for now as some lines below we already do a sync dialog too.

File: src/libsync/networkjobs.cpp [hunk 1]
```diff
@@ -447,7 +447,7 @@ bool CheckServerJob::finished()
     }
 
     bool success = false;
-    QByteArray body = reply()->readAll();
+    QByteArray body = reply()->peek(1024);
```
Comment (ogoffart) [inline src/libsync/networkjobs.cpp]: 1024 bytes should be enough for every status.php  replies?

Comment (guruz) [inline src/libsync/networkjobs.cpp]: Not sure. Right now we're at around 110 bytes. Shall I increase it or not?

**Comment by guruz** [comment]
> Why do you show a messagebox instead of displaying the error within the wizard?

No need to mess with the old wizard logic until @danimo has given us his code for the new wizard.

**Pull request merged**